app/components/system_message/index.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import CombinedJoinLeaveMessage from '../combined_joinleave_message/';
```

Comment:
These imports should be done with absolute imports when they're not in the same folder